browser_patches/webkit/patches/bootstrap.diff

```diff
@@ -1,8 +1,8 @@
 diff --git a/Source/JavaScriptCore/CMakeLists.txt b/Source/JavaScriptCore/CMakeLists.txt
-index 3566ca4859e09aaddaa7f97175a4b12f902780fc..ebab5e27a3dd4760291acaf74a5db117539fa8d8 100644
+index ad391d29fbf3..7361fa249d56 100644
```

Comment:
I think your git settings are off - we end up with all hashes changing their format.

Reply:
ah, i was about to update the patch just as you merged it :) well, i'll use the full format next week

Reply:
@q66 which Git version are you using and do you have the option `core.abbrev` in the config set? Curious to find out why it's only using at your machine the short ones.

Reply:
2.30.2 (on ubuntu), and it's unset

Reply:
are you using the export.sh script? This should automatically give you full-indexes since we set the override there.

(if you are using it, a config dump via `git config --list` would be helpful to debug it further.)